src/geometry/ArborX_Box.hpp

```diff
@@ -18,6 +18,9 @@
 #include <ArborX_Point.hpp>
 
 #include <Kokkos_Macros.hpp>
+#if __has_include(<Kokkos_ReductionIdentity.hpp>) // since Kokkos 4.0
```

Comment:
```suggestion
#if __has_include(<Kokkos_ReductionIdentity.hpp>) // FIXME remove once require Kokkos 4.0
```

Reply:
How strong do you feel about this because I think my comment was good enough and I don't like the suggested change

Reply:
I'm fine either way. I just wanted to have a simple way to stumble upon it in the future, so feel like adding a FIXME would be nice.

Reply:
Done